TODO

```diff
@@ -1,8 +1,6 @@
 - instead of using the shell in the service file, consider calling the generator
   binary again and doing the initalization from it.
 - logging in generators needs to go to kmsg, println!() is not enough.
-- add a testing mode where, if $ZRAM_GENERATOR_ROOT=... is set, and do everything
-  relative to this path then.
```

Comment:
There seems to have been a misunderstanding. This *does* say "everything", but actually the device paths and /proc file paths should not be touched.

Reply:
I took it the way that everything should be in that root, incl meminfo so that it is easy to write unit tests because it would be matter of just generating meminfo and configuration files in some temporary directory. What should that option do then?

Reply:
After thinking about this, the approach in this patch seems OK.